Units/parser-c.r/bit_field.c.d/args.ctags

```diff
@@ -1 +1,2 @@
 --sort=no
+--fields=+{c.end}
```

Comment:
Could you try to use wildcard notation `{end}` here? I hope this notation works well.
`{end}` means `{*.end}`. And `*` means "any language".

I implemented but I don't have a test case for the wildcard notation.
I would like to reuse this as a test case for the notation.